src/activities/assignments/AssignmentEntity.js

```diff
@@ -144,6 +144,133 @@ export class AssignmentEntity extends Entity {
 		return entity ? entity.properties.url : undefined;
 	}
 
+	/**
+	 * @returns {string} Name of the selected group category for the assignment type
+	 */
+	getAssignmentTypeSelectedGroupCategoryName() {
+		if (!this._entity) return null;
+		const subEntity = this._entity.getSubEntityByRel(Rels.Assignments.folderType);
+		if (!subEntity || !subEntity.properties || !subEntity.properties.groupName) return null;
+		return subEntity.properties.groupName;
+	}
+
+	/**
+	 * @returns {bool} If the assignment type cannot be changed
+	 */
+	isAssignmentTypeReadOnly() {
```

Comment:
Should this maybe default to `true` given that it is the "you can't do a thing" case?